rayon-core/src/lib.rs

```diff
@@ -36,7 +36,9 @@ mod unwind;
 mod util;
 
 pub use configuration::Configuration;
+pub use configuration::ExitHandler;
```

Comment:
nit: you made the `rayon-core/src/lib.rs` and `test.rs` files executable -- please revert that.